chcemvediet/apps/inforequests/admin.py

```diff
@@ -319,10 +319,18 @@ def get_queryset(self, request):
     def get_inforequest(self, obj):
         return obj.inforequest
 
-    def has_delete_permission(self, request, obj=None):
-        if obj is None:
-            return True
-        return not obj.is_main
+    def delete_constraints(self, obj):
```

Comment:
@martinmacko47 Nie je to uplne potrebne, ale chcel som to spravit rovnako ako v Action. Potom ked spravim admin mazanie pre vsetky instancie, pozriem co vsetko sa opakuje a tie casti kosu by som dal do spolocnej classy. Rovnako aj template.

Reply:
Ano, bolo by dobre spravit casom nejaky mixin, napriklad `DeleteConstraintsAdminMixin`, ktory definuje prazdnu abstraktnu metodu `delete_constraints` a pouzije ju v `render_delete_form` a `delete_model`, nech netreba tu logiku opakovat.